smart-contract-verifier/smart-contract-verifier-server/src/types/verify_response.rs

```diff
@@ -84,16 +84,17 @@ impl VerifyResponseOk for SourcifySuccess {
 }
 
 impl VerifyResponseWrapper {
-    pub fn ok<T: VerifyResponseOk>(success: T) -> Self {
+    pub fn ok<T: VerifyResponseOk>(
+        success: T,
+        post_action_responses: Option<PostActionResponses>,
+    ) -> Self {
         let (source, extra_data) = success.result();
         VerifyResponse {
             message: "OK".to_string(),
             status: Status::Success.into(),
             source: Some(source),
             extra_data: Some(extra_data),
-            post_action_responses: Some(PostActionResponses {
-                lookup_methods: None,
-            }),
+            post_action_responses,
```

Comment:
I suggest to make the response to contain a valid PostActionResponses structure, with all action results set to be None even if there were no additional actions. In general, it's more like a prost specifics that make custom messages to be optional. In the proto definition the `post_action_responses` is not optional, though

Reply:
Btw, what should we do with sourcify and vyper verifiers, then? Should we validate that incoming requests have no `post_actions` specified or ignore them and just always send response with `None`s

Reply:
I'm probably for ignoring those actions and returning `None`s

Reply:
Fixed